source/documentation/getting-started/kubectl-config.html.md.erb

```diff
@@ -39,8 +39,9 @@ You will need to:
 
 You can request access to the `ministryofjustice` GitHub organisation by:
```

Comment:
Thinking maybe something like:

If you have a `@digital.justice.gov.uk` or `@justice.gov.uk` email address you can [join the `ministryofjustice` GitHub organisation via single sign-on (SSO)](https://github.com/orgs/ministryofjustice/sso).

If you cannot use SSO please contact the #ask-operations-engineering slack channel or via [operations-engineering@digital.justice.gov.uk](mailto:operations-engineering@digital.justice.gov.uk)